apps/src/lab2/levelEditors/levelData/EditMusicLevelData.tsx

```diff
@@ -236,13 +236,19 @@ const EditMusicLevelData: React.FunctionComponent<EditMusicLevelDataProps> = ({
         </div>
       </CollapsibleSection>
       <hr />
+      <div>
+        {'You can also edit toolbox blocks using Blockly using Extra Links.'}
```

Comment:
Could we add a button here that takes users to toolbox mode as we do in legacy edit mode?

Reply:
Will consider as follow-up!